docs/src/pages/[platform]/connected-components/authenticator/advanced/useAuthenticator.angular.mdx

```diff
@@ -1,7 +1,7 @@
 import { Alert } from '@aws-amplify/ui-react';
 
 <Alert
-  variation="info"
+  variation="warning"
```

Comment:
Upgraded to a warning to be more prevalent:

![image](https://user-images.githubusercontent.com/6165315/229938331-adf43334-6011-43d1-bbea-4688342cdad6.png)